subprojects/docs/src/samples/kotlinDsl/androidSingleBuild/src/androidTest/java/org/gradle/kotlin/dsl/samples/androidstudio/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package org.gradle.kotlin.dsl.samples.androidstudio
+
+import android.support.test.InstrumentationRegistry
+import android.support.test.runner.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
```

Comment:
`Instrumented test; will execute on an Android device.`

Or:
`An instrumented test, which will execute on an Android device.`

Reply:
That comes from Android Studio default project and isn't displayed in the documentation. Won't fix.